Application/EdFi.Ods.Common/Configuration/ScheduledJobSetting.cs

```diff
@@ -0,0 +1,17 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.Ods.Common.Configuration;
+
+public class ScheduledJobSetting
+{
+    public string Name { get; set; }
+
+    public bool IsEnabled { get; set; }
+        
+    public double StartAtMinutes { get; set; }
```

Comment:
What does `StartAtMinutes` mean exactly? Would `PeriodMinutes` be a clearer name? That would be more consistent with the `PeriodicTimer` class that was added in .NET 6.